inscriptions/ISic001401.xml

```diff
@@ -190,19 +195,18 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI140909">
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#AL">
                 <ab>
-                    <lb n="1"/>
-
-
-                    Βετία Κο<supplied reason="omitted">ρν</supplied>η-<lb n="2"/>λία εὐσεβεστ<supplied reason="omitted">ά</supplied>-<lb n="3"/>τ<supplied reason="omitted">η</supplied>, ἔζησε ∙ ἔτ
-                    <supplied reason="omitted">η</supplied>
-                    <lb n="4"/> νʹ.
-                    <lb n="5"/>
+                    <lb n="1"/><persName type="attested"><name type="nomen">Βετία</name> <space quantity="1" unit="character"/> <name type="cognomen" cert="low">Κο<choice><corr>ρν</corr><sic>μ</sic></choice>η
+                    <lb n="2" break="no"/>λία</name></persName> <space quantity="1" unit="character"/> εὐσεβεστ<choice><corr>ά</corr><sic>ε</sic></choice>
```

Comment:
On the other hand, I would here go for:
`<choice><orig>εὐσεβεστέτα</orig><reg>εὐσεβεστάτη</reg></choice>`
Since there seems little reason to doubt Ansaldi here (not trying to represent anything odd), and so it is more liikely to be a linguistic variation.